cmd/build/helmify/static/README.md

```diff
@@ -218,6 +218,9 @@ information._
 | audit.readinessTimeout                                     | Timeout in seconds for audit's readiness probe                                                                                                                                                                                                                                                 | `1`                                                                                                                                                                   |
 | audit.livenessTimeout                                      | Timeout in seconds for the audit's liveness probe                                                                                                                                                                                                                                              | `1`                                                                                                                                                                   |
 | audit.logLevel                                             | The minimum log level for audit, takes precedence over `logLevel` when specified                                                                                                                                                                                                               | `null`                                                                                                                                                                |
+| audit.enablePubsub                                         | (alpha) Enabled pubsub to publish messages                                                                                                                                                                                                                                                     | `false`                                                                                                                                                               |
```

Comment:
nit: can you pls make this consistent with other fields that currently have `(alpha feature)`. prepend (alpha) is better IMO.